test/digitise/index.test.js

```diff
@@ -108,4 +108,73 @@ experiment('digitise/index.js', () => {
       })
     })
   })
+
+  experiment('.getSchemaCategories', () => {
+    // Array of simplified schema, containing just a category name (taken from the current set of schema JSON files)
+    const schemaArray = [
+      { category: 'Minimum value condition' },
+      { category: 'Hands off flows/levels' },
+      { category: 'Augmentation and Compensation conditions' },
+      { category: 'Flow/level measurement devices' },
+      { category: 'Construction and notices' },
+      { category: 'Ongoing measurement and maintenance' },
+      { category: 'Recording' },
+      { category: 'Storage conditions' },
+      { category: 'Time Limiting - abstractions start' },
+      { category: 'Chemical conditions' },
+      { category: 'Land on which licence authorises use of water' },
+      { category: 'Point at which water must be returned' },
+      { category: 'Temporary licence provision' },
+      { category: 'Groundwater conditions' },
+      { category: 'Ground source heating and cooling pumps' },
+      { category: 'Screening' },
+      { category: 'Fish and Eel passage' },
+      { category: 'Hydropower' },
+      { category: 'Impounding conditions' },
+      { category: 'Obstructing or impeding the flow of an inland water' },
+      { category: 'Removal of impounding works' },
+      { category: 'Water Rights Trading' },
+      { category: 'Environmental monitoring' },
+      { category: 'Control on abstraction' },
+      { category: 'Derogation Agreement' }
+    ]
+
+    // What the slugs should be for the categories, based on passing them through the slugify npm package
```

Comment:
As we're removing slugify with this change the comment no longer applies.

Reply:
Ah that's a new comment! It was just to explain where the slugs have come from. I can either expand it (to explain that we previously used slugify) or remove it completely? Any preference?

Reply:
Keep it, but yep, perhaps mention we used to use slugify and now we have our own implementation.